app/components/ui/Tooltip.tsx

```diff
@@ -1,79 +1,29 @@
-import * as Tooltip from '@radix-ui/react-tooltip';
-import { forwardRef, type ForwardedRef, type ReactElement } from 'react';
+import { type ReactNode } from 'react';
+import * as TooltipPrimitive from '@radix-ui/react-tooltip';
 
-interface TooltipProps {
```

Comment:
I appreciate this update but is this change necessary, or required for your UI, if not I recommend not do this change and if its required then keep name of the component same as that might confuse others who are already working on other prs. this can cause lots of conflicts also